resources/keda.yaml

```diff
@@ -8949,6 +8965,22 @@ spec:
       serviceAccountName: keda-operator
       terminationGracePeriodSeconds: 10
 ---
+apiVersion: monitoring.coreos.com/v1
+kind: PodMonitor
+metadata:
+  name: keda-operator
+  namespace: openshift-keda
+  labels:
+    name: keda-operator
+spec:
+  selector:
+    matchLabels:
+      name: keda-operator
+  podMetricsEndpoints:
+    - targetPort: 8080
+      path: /metrics
+      interval: 60s
+---
```

Comment:
@joelsmith I did not add the proposed PodMonitor objects for `custom-metrics-autoscaler-operator`  here as it is not keda related. Do you anticipate doing a replacement in  [mid-stream ](https://gitlab.cee.redhat.com/custom-metrics-autoscaler/cpaas-midstream/-/blob/ocma-2-rhel-8/distgit/containers/custom-metrics-autoscaler-operator/Dockerfile.in#L17) instead?



```
apiVersion: monitoring.coreos.com/v1
kind: PodMonitor
metadata:
  name: custom-metrics-autoscaler-operator
  namespace: openshift-keda
  labels:
    name: custom-metrics-autoscaler-operator
spec:
  selector:
    matchLabels:
      name: custom-metrics-autoscaler-operator
  podMetricsEndpoints:
  - targetPort: 8080
    path: /metrics
    interval: 60s
```

Reply:
I think I see why you're asking this... since all the resources here are for the CMA/KEDA OLM operator managing the other two controllers. But, I think this is the right place for it. The reason is that this is a case of the OLM operator managing itself (by telling Prometheus to monitor it) kind of like how it manages itself by telling OLM to install and run itself via the CSV.

My preferences is to keep the midstream repo small with mostly only build things and things directly connected to CPaaS and brew, and keep code and manifests here. There are some parts of the midstream which might edit the manifests from these repos, etc. but other than those edits, the rest of the code and manifests live here.

Reply:
Makes sense, I will adjust accordingly.